code/modules/atmospherics/machinery/components/trinary_devices/filter.dm

```diff
@@ -41,6 +41,7 @@
 
 /obj/machinery/atmospherics/components/trinary/filter/update_overlays()
 	. = ..()
+	cut_overlays()
```

Comment:
Aren't these noops?
![image](https://user-images.githubusercontent.com/58055496/109477912-7467aa80-7a2d-11eb-8f69-75de240abcf2.png)

Reply:
is cut_overlays() lmao
![image](https://user-images.githubusercontent.com/42839747/109481596-3d20ea00-7a7d-11eb-8891-52d2f28b7e1b.png)